src/free-style.ts

```diff
@@ -170,10 +170,11 @@ function stringifyProperties (properties: Properties) {
 
   for (const [name, value] of properties) {
     if (value != null) {
-      if (Array.isArray(value)) {
-        result.push(value.filter(x => x != null).map(x => styleToString(name, x)).join(';'))
+      const value2 = typeof value === 'function' ? value() : value
```

Comment:
I'll tidy it up, but initial issue is that `value2` may be `null`. Should probably be `const value = typeof prop === 'function' ? prop() : prop` before checking the value.